tests/console/cups.pm

```diff
@@ -0,0 +1,73 @@
+# SUSE's openQA tests
+#
+# Copyright 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test basic capabilities of cups
+# Maintainer: Jan Baier <jbaier@suse.cz>
+
+use 5.018;
```

Comment:
why you need this ?

Reply:
That's probably copy&paste from some kernel/ltp test.

Reply:
please remove it